extensions/arc/src/models/postgresModel.ts

```diff
@@ -53,10 +53,8 @@ export class PostgresModel extends ResourceModel {
 
 	/** Returns the major version of Postgres */
 	public get engineVersion(): string | undefined {
-		const kind = this._config?.kind;
-		return kind
-			? kind.substring(kind.lastIndexOf('-') + 1)
-			: undefined;
+		const kind = this._config?.spec.engine.version;
+		return kind;
```

Comment:
Since we don't have to substring it anymore, this function can just `return this._config?.spec.engine.version;`